ivy/functional/frontends/tensorflow/func_wrapper.py

```diff
@@ -215,6 +215,7 @@ def _wrap_raw_ops_alias(fn: callable, kw_update: Dict) -> callable:
             )
             new_params.append(param.replace(name=name, kind=param.KEYWORD_ONLY))
         new_signature = sig.replace(parameters=new_params)
+        new_module = inspect.stack()[2][0].f_globals["__name__"]
```

Comment:
What does this line here exactly do? it seems unclear, is it retrieving the current module?